R/output-ipc.R

```diff
@@ -173,3 +176,44 @@ dataframe__write_ipc <- function(
   })
   invisible(NULL)
 }
+
+#' Write to Arrow IPC Streaming Format
+#'
+#' @inheritParams rlang::args_dots_empty
+#' @inheritParams lazyframe__sink_ipc
+#' @inherit dataframe__write_parquet return
+#' @examplesIf requireNamespace("nanoarrow", quiet = TRUE) && nanoarrow::nanoarrow_with_zstd()
+#' tmpf <- tempfile(fileext = ".arrows")
```

Comment:
`.arrow` or `.arrows`? `.arrows` is used here and in the README, but `.arrow` is used in other examples

Reply:
Whitch README?

`.arrow` is for the Arrow file.
`.arrows` is for the Arrow Stream.

Reply:
In fact, file extensions are not used very often correctly.
I've seen people confuse `.arrow` with `.arrows` many times.